requirements/README.md

```diff
@@ -1,36 +1,15 @@
-The requirements.txt and requirements_ansible.txt files are generated from requirements.in and requirements_ansible.in, respectively, using `pip-tools` `pip-compile`. The following commands should do this if ran inside the tools_awx container.
+The `requirements.txt` and `requirements_ansible.txt` files are generated from `requirements.in` and `requirements_ansible.in`, respectively, using `pip-tools` `pip-compile`.
 
-Run these commands from the root of the awx repo. This will produce python 3 requirements files.
+Run `./upgrade.sh` command from inside `./requirements` directory of the awx repository.
```

Comment:
this line gives `./upgrade.sh`, but the file appears to be named `./updater.sh`

Reply:
Omg, thank you. Fixed/forced pushed it. I'm also open to renaming the script to something more meaningful if there is any suggestion.